documentation/Installation.md

```diff
@@ -1437,6 +1438,54 @@ services:
     cgroupDriver: systemd
 ```
 
+#### kubeadm_patches
+
+*Installation task*: `deploy.kubernetes`
+
+*Can cause reboot*: no
+
+*Can restart service*: always yes, `kubelet`
+
+*OS specific*: No
+
+In `services.kubeadm_patches` section you can override control-plane pod settings as well as kubelet settings on per node basis.
+This feature is implemented by using of [kubeadm patches](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/control-plane-flags/#patches).
+
+**Note**: `patches` feature is available for control-plane pods (etcd, kube-apiserver, kube-controller-manager, kube-scheduler) since Kubernetes **v1.22.0** and for kubelet since Kubernetes **v1.25.0**.
+
+The following is an example of control-plane settings override:
+```
+services:
+  kubeadm_patches:
+    apiServer:
+      all_nodes:
+        max-requests-inflight: 500
+      master-3:
+        max-requests-inflight: 600
+    etcd:
+      master-1:
+        snapshot-count: 110001
+      master-2:
+        snapshot-count: 120001
+      master-3:
+        snapshot-count: 130001
+    controllerManager:
+      all_nodes:
+        authorization-webhook-cache-authorized-ttl: 30
+    scheduler:
+      master-2:
+        profiling: true
+    kubelet:
+      worker5:
+        maxPods: 100
+      worker6:
+        maxPods: 200
+```
```

Comment:
Probably makes sense to use approach similar to `thirdparties`? For example:
```yaml
services:
  kubeadm_patches:
  - type: apiServer
    node: master-1
    patch:
      max-requests-inflight: 500
  - type: etcd
    nodes: master-1,master-2
    patch:
      snapshot-count: 110001
  - type: scheduler
    group: master
    patch:
      profiling: true
  - type: kubelet
    group: all
    patch:
      maxPods: 200
```
Important detail is that `balancer` group not permitted and not included in `all`.

This API is not only similar to thirdparties, it also supports whole groups like `master` or `worker`. It also allows to combine several equal `node` patches into one `nodes` patch (multi-node)

Reply:
Some plugins' installation procedure types also have the similar API, but 1) `all` is not supported in both; 2) there is `groups` instead of `group` and they accept list, not string. The same for `nodes`:
```yaml
groups:
  - control-plane
  - worker
nodes:
  - control-plane-1
  - control-plane-2
```

Reply:
fixed:
```
services:
  kubeadm_patches:
    apiServer:
      - groups: [control-plane]
        patch:
          max-requests-inflight: 500
      - nodes: [master-3]
        patch:
          max-requests-inflight: 600
```